signer/signer.go

```diff
@@ -23,6 +24,13 @@ import (
 // signingAgent is the unprotected header field used by signature.
 const signingAgent = "Notation/1.0.0"
 
+var (
+	reservedAnnotationPrefixes = []string{
+		"org.opencontainers",
```

Comment:
Why `org.opencontainer` is a reserved prefix?

Users may want those [standard annotations](https://pkg.go.dev/github.com/opencontainers/image-spec/specs-go/v1#pkg-constants).